sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ Caller must provide one of the following:
  (3) only keyword `continuation_token` to restart a poller from a saved state.
 -  `list_all_orders` renamed to `list_all_reservations`.
 
+##### `IdentityClient`
+- Added support for Azure Active Directory authentication for the Identity client
```

Comment:
Lets format the changelog according to the guide: https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance

The changes for both `IdentityClient` and `PhoneNumberAdministrationClient` should be under the "Breaking Change" or "Added"